docs/visual-basic/language-reference/operators/isnot-operator.md

```diff
@@ -18,26 +18,46 @@ result = object1 IsNot object2
 ```
 
 ## Parts
- `result`
- Required. A `Boolean` value.
 
- `object1`
- Required. Any `Object` variable or expression.
+- `result`
```

Comment:
These `-` don't follow the convention of the other articles, so we should remove them.

Reply:
@adegeo I changed some articles to use list item a long time ago :cry:. So there will still be some articles that don't follow the convention of others.